server/application-server/src/main/java/de/tum/in/www1/hephaestus/EnvConfig.java

```diff
@@ -0,0 +1,17 @@
+package de.tum.in.www1.hephaestus;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "environment")
+public class EnvConfig {
```

Comment:
You can remove the file and use `@Value` in the service for example.

Articel: 
https://www.baeldung.com/spring-value-annotation